modules/nf-core/btyper3/tests/main.nf.test

```diff
@@ -0,0 +1,84 @@
+nextflow_process {
+
+    name "Test Process BTYPER3"
+    script "../main.nf"
+    process "BTYPER3"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "btyper3"
+
+    test("sarscov2 - fasta") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + '/genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.tsv,
+                    process.out.versions
+                    ).match()
+                }
```

Comment:
Can you also assert whether the results folder is present? maybe even check the subfiles that are created in that folder.